CHANGELOG.md

```diff
@@ -12,10 +12,12 @@ _This release is scheduled to be released on 2024-01-01._
 ### Added
 
 - Added node 21 to the test matrix
+- Added transform object to calendar:customEvents
 
 ### Removed
 
 - Removed Codecov workflow (not working anymore, other workflow required) (#3107)
+- Removed titleReplace from calendar, replaced + extended by customEvents (backward compatibility included)
```

Comment:
as titleReplace is not completely removed (backward compatibility is included in PR) but users should be encouraged to use the new style, I'm not sure if this is the right place. Perhaps you find a better one

Reply:
this would probably best be documented in the [doc-repo](https://github.com/MichMich/MagicMirror-Documentation) after this PR got merged